front/lib/constants/errors.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-magic-numbers */
+export const HTTP_ERRORS = {
+  UNAUTHORIZED: 401,
+  FORBIDDEN: 403,
+  NOT_FOUND: 404,
+  DUPLICATE: 409,
+  UNPROCESSABLE_ENTITY: 422,
+  INTERNAL_SERVER_ERROR: 500
+}
+
+export const NOT_FOUND = {
```

Comment:
Can these errors be translated ?

Reply:
Same, we'll move this in an issue to be thought about later on.

Reply:
I'll remove these files for now as we don't use any of the variables